### PR TITLE
MAINT-28485: Make sure that only the activity owner has the right to edit it and that administrators can only delete it.

### PR DESCRIPTION
**Pull request by mdiha**
I made sure that only the activity's owner is allowed to edit it but instead platform administrators have only the right to delete t.

### CODE REVIEW

File: component/webui/src/main/java/org/exoplatform/social/webui/activity/BaseUIActivity.java [hunk 1]
```diff
@@ -577,8 +578,9 @@ public boolean isActivityDeletable() throws SpaceException {
       }
     }
 
+    UserACL userACL = getApplicationComponent(UserACL.class);
     if (space != null) {
-      return spaceService.isManager(space, Utils.getOwnerRemoteId());
+      return userACL.isSuperUser() || spaceService.isManager(space, Utils.getOwnerRemoteId()) || ConversationState.getCurrent().getIdentity().getGroups().contains(userACL.getAdminGroups());
```
Comment (ahamdi) [inline component/webui/src/main/java/org/exoplatform/social/webui/activity/BaseUIActivity.java]: We need to check if the user is a super manager (member of space administration group)
```suggestion
      return userACL.isSuperUser() || spaceService.isManager(space, Utils.getOwnerRemoteId()) || spaceService.isSuperManager(Utils.getOwnerRemoteId())
```

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: component/webui/src/main/java/org/exoplatform/social/webui/activity/BaseUIActivity.java (hunk 1 above)
Comment (mdiha) [reply component/webui/src/main/java/org/exoplatform/social/webui/activity/BaseUIActivity.java]: Okay i will add it

File: component/webui/src/main/java/org/exoplatform/social/webui/activity/BaseUIActivity.java [hunk 2]
```diff
@@ -577,8 +578,9 @@ public boolean isActivityDeletable() throws SpaceException {
       }
     }
 
+    UserACL userACL = getApplicationComponent(UserACL.class);
     if (space != null) {
-      return spaceService.isManager(space, Utils.getOwnerRemoteId());
+      return userACL.isSuperUser() || spaceService.isSuperManager(Utils.getOwnerRemoteId()) || spaceService.isManager(space, Utils.getOwnerRemoteId()) || ConversationState.getCurrent().getIdentity().getGroups().contains(userACL.getAdminGroups());
```
Comment (ahamdi) [inline component/webui/src/main/java/org/exoplatform/social/webui/activity/BaseUIActivity.java]: The check on platform administrators group is no more needed as we check on Space administration group

Comment (mdiha) [reply component/webui/src/main/java/org/exoplatform/social/webui/activity/BaseUIActivity.java]: Done

**Review by ahamdi — APPROVED** [review]

**Pull request merged**